pkg/endpointslices/endpointslices.go

```diff
@@ -35,9 +35,9 @@ func New(cs *client.ClientSet) (*EndpointSlicesExporter, error) {
 	if err != nil {
 		return nil, err
 	}
-	nodesRolesMap := map[string][]string{}
+	nodesRolesMap := map[string]string{}
 	for _, node := range nodeList.Items {
-		nodesRolesMap[node.Name] = types.GetNodeRoles(&node)
+		nodesRolesMap[node.Name] = types.GetNodeRole(&node)
```

Comment:
we dont need to check the output if it not ""?

Reply:
I also think we should check the output is not "".